receiver/splunkenterprisereceiver/scraper.go

```diff
@@ -27,10 +27,11 @@ import (
 var errMaxSearchWaitTimeExceeded = errors.New("maximum search wait time exceeded for metric")
 
 type splunkScraper struct {
-	splunkClient *splunkEntClient
-	settings     component.TelemetrySettings
-	conf         *Config
-	mb           *metadata.MetricsBuilder
+	splunkClient  *splunkEntClient
+	settings      component.TelemetrySettings
+	conf          *Config
+	mb            *metadata.MetricsBuilder
+	failedScrapes int64
```

Comment:
Perhaps this is potentially safer as an atomic int to allow for concurrent writes?

Reply:
Hm, I'm not sure that's necessary because the receiver is only running a scrape cycle one time per minute, and we wait for all scrapes to be concluded before incrementing this.

Reply:
I went ahead and added this.